src/api/config/feature.yml

```diff
@@ -1,30 +1,20 @@
-production:
-  features: &default
-    image_templates: true
-    kiwi_image_editor: false
-    cloud_upload: false
-    cloud_upload_azure: false
-    bootstrap: false
+# production defaults are in lib/feature_switch/obs_repository.rb
+# you can overwrite them here though
```

Comment:
no you can't. as we will update the package and you're changes are gone...

also expecting admins to change ruby code to configure the app is not what we want.

Reply:
the config file is config(noreplace) in the package and that's where admin should configure it. I'm not saying admins should change ruby code - only to read if they care. If the comment is misleading, do you have suggestions how to make it clearer?

Reply:
repeat the defaults as comment?

Reply:
I did that now - please check if that fits your expectations